src/gui/folder.cpp

```diff
@@ -615,6 +615,39 @@ void Folder::slotWatchedPathChanged(const QString &path, ChangeReason reason)
     scheduleThisFolderSoon();
 }
 
+void Folder::slotFilesLockReleased(const QSet<QString> &files)
+{
+    qCInfo(lcFolder) << "Going to unlock office files" << files;
+
+    for (const auto &file : files) {
+        const auto fileRecordPath = fileFromLocalPath(file);
+        SyncJournalFileRecord rec;
+        const auto canUnlockFile = journalDb()->getFileRecord(fileRecordPath, &rec)
+            && rec.isValid()
+            && rec._lockstate._locked
+            && rec._lockstate._lockOwnerType == static_cast<qint64>(SyncFileItem::LockOwnerType::UserLock)
+            && rec._lockstate._lockOwnerId == _accountState->account()->davUser();
+
+        if (!canUnlockFile) {
+            qCInfo(lcFolder) << "Skipping file" << file << "with rec.isValid():" << rec.isValid()
+                             << "and rec._lockstate._lockOwnerId:" << rec._lockstate._lockOwnerId << "and davUser:" << _accountState->account()->davUser();
+            continue;
+        }
+        const QString remoteFilePath = remotePathTrailingSlash() + rec.path();
+        qCInfo(lcFolder) << "Unlocking an office file" << remoteFilePath;
+        _officeFileLockReleaseUnlockSuccess = connect(_accountState->account().data(), &Account::lockFileSuccess, this, [this, remoteFilePath]() {
```

Comment:
@mgallien I disagree. This is rather an unusual format. Most of our code does not have that space as well as examples you find online. Is this a recommendation from the latest standard? Otherwise, I would keep it without the space